obs/specs/_service

```diff
@@ -0,0 +1,27 @@
+<services>
+    <service name="tar_scm" mode="disabled">
+        <param name="url">https://github.com/trento-project/trento.git</param>
+        <param name="scm">git</param>
+        <param name="revision">main</param>
+        <param name="exclude">.git</param>
+        <param name="exclude">.github</param>
+        <param name="extract">web/frontend/package.json</param>
+        <param name="extract">web/frontend/package-lock.json</param>
+        <param name="exclude">web/frontend/package-lock.json</param>
+        <param name="versionformat">0.4.0+%ct.%h</param>
```

Comment:
It's sedded as in https://confluence.suse.com/display/haslesforsap/Automatic+OBS+packages+updates+with+GitHub+and+GitHub+Actions  now

Reply:
> > > I also have one more question: how would we generate the OBS-compliant changelog?
> > 
> > 
> > By a shell script, I guess. I think we should address it in the next PR.
> 
> hey Aleksei, WRT this, maybe you can copy this:
> https://github.com/SUSE/sap_host_exporter/blob/master/.ci/gh_release_to_obs_changeset.py
> 
> Rest of the changes mostly LGTM, I will give it a closer look as soon as I can.

The problem with tockens is that the **$ obs token -o runservice** will fail, because the obs cannot download the vendors from outside.